buildSrc/src/main/kotlin/partiql.conventions.gradle.kts

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Already discussed this offline but adding it here -- is there a need for Plugin.conventions if every subproject uses it?

Reply:
Technically no, but I was following the Gradle rec https://docs.gradle.org/current/samples/sample_convention_plugins.html

I can apply the convention plugin to all sub-projects by default which would achieve the same thing

Reply:
Just kidding, the plugin isn't available in the `buildSrc/build.gradle.kts` so it needs to be applied in each project's build script